CONTRIBUTING.md

```diff
@@ -104,9 +104,9 @@ Crayons repository uses github pre commit hooks, follow the below steps to commi
 
 8. Give a commit message when it prompts.
 
-9. Run `git push -u origin <your-branch>`, to push your changes to the forked repo. git hooks will run all the test before pushing the changes. if any test fails, rewrite the tests and make sure all the test passes before pushing again.
+9. Run `git push -u origin <your-branch>`, to push your changes to the forked repo. git hooks will run all the test before pushing the changes. if any test fails, rewrite the tests and make sure all the tests pass before pushing again.
 
-10. Go to the forked repo in github and raise a PR. wPlease wait for the PR to be approved or address comments if any.
+10. Go to the forked repo in github and raise a PR. Please wait for the PR to be approved. If there are any comments, address them.
```

Comment:
Could we modify this as follows?
Navigate to the forked repo on GitHub and raise a PR. If there are any comments on your PR, address them. Please wait for the PR to be approved.